resources/assets/sass/_base.scss

```diff
@@ -0,0 +1,21 @@
+html {
+  @apply .m-0;
+}
+
+body {
```

Comment:
Is there any value gained from moving these classes to the scss instead of keeping them in the template?

Reply:
Took some time to think about this one. I don't think there's a benefit to building out individual Sass files, since it goes against the flow of using tailwind. My inclination is to reach for a class and abstract when it comes to styles. 

I'll move these back to the DOM.